### DISCUSSION TITLE
Spacy-stanza support in v3.0

### DISCUSSION THREAD

**Discussion by buhrmann** [explosion/spaCy#7037] [Help: Coding & Implementations]
Hi, I was just wondering if you have an approx. ETA for updating spacy-stanza to v3? I've seen there is already a PR, but not sure if it's a priority for you guys at all?

**Comment by adrianeboyd** [comment]
We are working on it, but don't have a planned release date at this point.

If you'd like you can try out PR [#58](https://github.com/explosion/spacy-stanza/pull/58) and leave a comment if you run into any bugs. I expect the final version to be very similar, but there may be some minor changes to the API before it's merged.

**Reply by buhrmann** [reply]
Ok, no problem. That's in fact what I'm doing right now (trying out PR #58).

I've found one problem so far and left a comment in your PR (it should be `"dir"` rather than `"model_dir"`).

I'll let you know if I find anything else.
